core/panic-handler/src/lib.rs

```diff
@@ -149,13 +149,13 @@ fn panic_hook(info: &PanicInfo, report_url: &'static str, version: &str) {
 
 	let mut stderr = io::stderr();
 
-	let _ = writeln!(stderr, "");
```

Comment:
It's debatable whether this is a positive change. The ` ""` make it align nicely.